recipe/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "titus2" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd7ca43e2a7cde530e0ae440cb65d87aac9eedd8cdba9c47cff5d1882e88c297
+  patches:
+    - patches/0001-skip-test-for-darwin.patch  # [osx]
+
+build:
+  skip: True  # [py>=310]
+  # missing win build for avro --> avro-python3
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+
+requirements:
+  build:
+    - patch  # [osx]
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - numpy >=1.15.0,<1.20.0
+    - avro-python3 >=1.8.2
+    - ply >=3.11
+    - pyyaml >=5.1.2
+    - pytz >=2019.1
+
+test:
+  imports:
+    - titus
+    - titus.producer
+    - titus.lib
+    - titus.lib.model
+    - titus.lib.prob
+    - titus.lib.stat
+    - titus.inspector
+  requires:
+    - pip
+  source_files:
+    - test
+  commands:
+    - pip check
+    - python -m unittest discover -s test -p "test*.py"
```

```suggestion
    - pip
    - pytest
  source_files:
    - test
  commands:
    - pip check
    - pytest test -o python_files=test*.py -k "not testUnpackBoolean" -v  # [osx]
    - pytest test -o python_files=test*.py -v  # [not osx]
```

Pytest can skip tests from the command line. Easier to maintain than patching test files directly.